x-pack/legacy/plugins/infra/server/lib/adapters/metrics/models/host/host_docker_top_5_by_cpu.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  InfraMetricModelCreator,
+  InfraMetricModelMetricType,
+  InfraMetricModel,
+} from '../../adapter_types';
+import { InfraMetric } from '../../../../../graphql/types';
+
+export const hostDockerTop5ByCpu: InfraMetricModelCreator = (
+  timeField,
+  indexPattern,
+  interval
+): InfraMetricModel => ({
+  id: InfraMetric.hostDockerTop5ByCpu,
+  requires: ['docker.cpu'],
+  index_pattern: indexPattern,
+  interval,
+  time_field: timeField,
+  type: 'timeseries',
+  series: [
+    {
+      id: 'avg-cpu',
+      metrics: [
+        {
+          field: 'docker.cpu.total.pct',
+          id: '61ca57f2-469d-11e7-af02-69e470af7417',
```

Comment:
Where does this hardcoded ID come from? How is it generated?

Reply:
Those are arbitrary, they are coming from TSVB. I usually change them to be a little more descriptive but forgot to in this case. They just need to be unique per metric array.